release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -341,6 +364,16 @@ This API still streamlines management of installed extensions, which includes Op
 * Better represents a more flexible packaging format
 * `Cluster` prefix clearly indicates that `ClusterExtension` objects are cluster-scoped, a change from {olmv0} where Operators could be either namespace-scoped or cluster-scoped
 
+For more information, see xref:../operators/olm_v1/arch/olmv1-operator-controller.adoc#olmv1-operator-controller[Operator Controller].
+
+[discrete]
+[id="ocp-4-16-olmv1-default-upgrade-constraints_{context}"]
+=== Default {olmv1} upgrade constraints changed to {olmv0} semantics (Technology Preview)
+
+In {product-title} 4.16, {olmv1-first} changes its default upgrade constraints from semantic versioning (semver) to {olmv0-first} semantics. Semver can be enabled by setting a feature gate.
```

Comment:
4.16.0? Wonder if you could say something like "Starting with {product-title} 4.16.0..."